docs/reference/configuration/raftstore-config.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Raftstore Configuration 
+summary: Learn about Raftstore configuration in TiKV.
+category: reference
+---
+
+# Raftstore Configurations
+
+Raftstore is TiKV's implementation of [Multi-raft](https://tikv.org/deep-dive/scalability/multi-raft/) to manage multiple Raft consensus groups on one node. Raftstore is comprised of two major components:
+
+- **Raftstore** component writes Raft logs into RaftDB.
+- **Apply** component resolves Raft logs and flush the data in the log into the underlying storage engine. 
+
+This section introduces the following features of Raftstore and their configurations:
+
+- [Multi-thread Raftstore](#multi-thread-raftstore)
+- [Hibernate Region](#hibernate-region)
+
+## Multi-thread Raftstore
+
+ Multi-thread support for the Raftstore and the Apply components means higher throughput and lower latency for standalone machines. In the multi-thread mode, each thread obtain peers from the queue in batch, so that small writes of multiple peers can be consolidated into a big write for better throughput.
```

Comment:
Standalone machines? As in single node deployments? Or for host machines?

Reply:
I guess it means a single node within a cluster. But will confirm with @BusyJay

Reply:
I think it means per host too, yes :)

Reply:
updated. PTAL again @Hoverbear